cmd/plugin.go

```diff
@@ -83,7 +83,7 @@ var (
 		Use:     "install",
 		Short:   "Install new plugin",
 		Args:    cobra.ExactArgs(2), // TODO: Can I check real git repo?
-		Example: "vatz plugin install github.com/dsrvlabs/<somewhere> name",
+		Example: "vatz plugin install <githubAddress> <pluginName>",
```

Comment:
@qkrybin 
I think right format one of followings
1. github.com/dsrvlabs/<somewhere> <pluginName> 
2. github.com/dsrvlabs/<somewhere> <pluginName> or <your plugin address> <pluginName>

Reply:
oh but the community plugin is not in dsrv repo, so I thought it would be better to write a generic example.

Reply:
> > oh but the community plugin is not in dsrv repo, so I thought it would be better to write a generic example.
> 
> okay, that make sense. how about 3rd option?, if you think still `"vatz plugin install <githubAddress> <pluginName>"` is good. I am good with it.

I changed it to 3rd one!